ccd-definition/CaseEvent.json

```diff
@@ -0,0 +1,202 @@
+    "DisplayOrder": 6,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42742,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "enterOthers",
+    "Name": "Enter others who need notice",
+    "Description": "Entering others for the case",
+    "DisplayOrder": 7,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42743,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "enterGrounds",
+    "Name": "Enter grounds for application",
+    "Description": "Entering the grounds for the application",
+    "DisplayOrder": 8,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42744,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "enterRiskHarm",
+    "Name": "Enter risk & harm to children",
+    "Description": "Entering opinion on risk and harm to children",
+    "DisplayOrder": 9,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42745,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "enterParentingFactors",
+    "Name": "Factors affecting parenting",
+    "Description": "Entering the factors affecting parenting",
+    "DisplayOrder": 10,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42746,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "enterInternationalElement",
+    "Name": "Enter international element",
+    "Description": "Entering the international element",
+    "DisplayOrder": 11,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42747,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "otherProceedings",
+    "Name": "Enter other proceedings",
+    "Description": "Entering other proceedings and allocation proposals",
+    "DisplayOrder": 12,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42748,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "otherProposal",
+    "Name": "Enter allocation proposal",
+    "Description": "Entering other proceedings and allocation proposals",
+    "DisplayOrder": 13,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": 42748,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "attendingHearing",
+    "Name": "Attending hearing",
+    "Description": "Enter extra support needed for anyone to take part in hearing",
+    "DisplayOrder": 13,
```

Comment:
There is an error here. It also present in the spreadsheet. Two 13s. attending the court is not show in the dropdown list

Reply:
this will effect all future numbers. Up to you whether this is fine for now and a fix will be done separately

Reply:
Will fix that separately @swalker125